eng/Versions.props

```
@@ -6,16 +6,25 @@
     Roslyn version
   -->
   <PropertyGroup>
+    <!-- All packages produced by the Roslyn official build should be suffixed with '-vs' to distinguish them from VMR produced packages -->
+    <RoslynPackageSuffix Condition="'$(DotNetBuildFromVMR)' != 'true'">-vs</RoslynPackageSuffix>
```

Consider prefixing this with `_` as "private" property.

```suggestion
    <_RoslynPackageSuffix Condition="'$(DotNetBuildFromVMR)' != 'true'">-vs</_RoslynPackageSuffix>
```